ketcher-autotests/tests/File-Management/Open-And-Save-Files/Rxn-Files/rxn-files.spec.ts

```diff
@@ -1,11 +1,277 @@
-import { expect, test } from '@playwright/test';
+import { expect, test, Page } from '@playwright/test';
 import {
   takeEditorScreenshot,
   receiveFileComparisonData,
   openFileAndAddToCanvas,
   saveToFile,
+  drawBenzeneRing,
+  getCoordinatesTopAtomOfBenzeneRing,
+  clickOnAtom,
+  clickOnBond,
+  BondTool,
+  selectNestedTool,
+  ArrowTool,
+  selectAtomInToolbar,
+  AtomButton,
+  BondType,
+  clickOnTheCanvas,
+  selectTopPanelButton,
+  TopPanelButton,
+  RgroupTool,
+  pressButton,
+  selectLeftPanelButton,
+  LeftPanelButton,
+  selectAction,
+  dragMouseTo,
+  setAttachmentPoints,
+  moveMouseToTheMiddleOfTheScreen,
+  getCoordinatesOfTheMiddleOfTheScreen,
 } from '@utils';
 import { getRxn } from '@utils/formats';
+import { drawReactionWithTwoBenzeneRings } from '@utils/canvas/drawStructures';
+
+async function savedFileInfoStartsWithRxn(page: Page, wantedResult = false) {
+  await selectTopPanelButton(TopPanelButton.Save, page);
+  const textareaSelector = 'textarea[class^="Save-module_previewArea"]';
+  const textareaElement = await page.$(textareaSelector);
+  const textareaText = await textareaElement?.textContent();
+  const expectedSentence = '$RXN';
+  wantedResult
+    ? expect(textareaText?.startsWith(expectedSentence)).toBeTruthy()
+    : expect(textareaText?.startsWith(expectedSentence)).toBeFalsy();
+}
+
+test('Open and Save file - Reaction with atom and bond properties', async ({
+  page,
+}) => {
+  /**
+   * Test case: EPMLSOPKET-1897
+   * Description: Reaction with atom and bond properties
+   */
+  const xOffsetFromCenter = 40;
+  await page.goto('');
+  await drawBenzeneRing(page);
+  await selectNestedTool(page, BondTool.SINGLE_AROMATIC);
+  await clickOnBond(page, BondType.DOUBLE, 1);
```

Comment:
Does it make sense to add more context to the numbers in lines: 55, 57, 59, 61, 90, 94?

Reply:
55, 57, 59, 90 - are indexes of atoms and bonds which need to be clicked which is kinda particular and expected because at the beginning of the test we always create same structure. I do not see how to add more clarity here if you have any suggestions - I'm totally on board.
61 and 94 are zero point (center) for Y axis.